### PR TITLE
feat/GRW-991/CPAS design fixes

### PR DESCRIPTION
**Pull request by marian-buc**
## What?

- Added padding on semi-small screens
- Breakpoint moved to 840px
- fixed X-axis layout
- added grey200 bg < 800px
- added extra padding to InnerWrapper on CheckoutPayment on medium screens

## Why?
CPAS rollout

**Ticket(s): [GRW-991]**

image
image

image
image

[GRW-991]: https://hedvig.atlassian.net/browse/GRW-991?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx [hunk 1]
```diff
@@ -87,7 +94,7 @@ export const CheckoutPageWrapper = ({
           </LogoLink>
         )}
         <BackButton onClick={handleClickBackButton} />
-        <InnerWrapper>{children}</InnerWrapper>
+        <InnerWrapper className={className}>{children}</InnerWrapper>
```
Comment (dendoesit) [inline src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx:97]: why do we need the className here ?

File: src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx [hunk 2]
```diff
@@ -44,9 +48,12 @@ const Wrapper = styled.div`
 const InnerWrapper = styled.div`
   margin: 0 auto;
   position: relative;
+  background: ${colorsV3.gray200};
+  padding: 0rem 1rem;
```
Comment (dendoesit) [inline src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx:52]: this adds extra padding that we should not have. 
you can see the design here that left right we have 16px on both screens. 
https://www.figma.com/file/GQ1b2GBtO3SxGW2YaH7qng/Purchase-Flow-Checkout?node-id=327%3A1852

Comment (marian-buc) [reply src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx:52]: oh, seems like the issue is on CheckoutPayment only

File: src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx (hunk 1 above)
Comment (marian-buc) [reply src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx:97]: to be able to pass custome styles such as CheckoutPaymentWrapper

File: src/client/utils/mediaQueries.ts [hunk 3]
```diff
@@ -1,11 +1,13 @@
 export const BREAKPOINTS = {
   largeScreen: 1000,
+  mediumLargeScreen: 840,
   mediumScreen: 800,
   mediumSmallScreen: 450,
   smallScreen: 375,
 }
 
 export const LARGE_SCREEN_MEDIA_QUERY = `@media screen and (min-width: ${BREAKPOINTS.largeScreen}px)`
+export const MEDIUM_LARGE_SCREEN_MEDIA_QUERY = `@media screen and (min-width: ${BREAKPOINTS.mediumLargeScreen}px)`
```
Comment (robinandeer) [inline src/client/utils/mediaQueries.ts:10]: Do we really need a new breakpoint? 🤔

**Review by robinandeer — COMMENTED** [review]
Can you include some screenshots of the changes? Easier to review :)

File: src/client/utils/mediaQueries.ts (hunk 3 above)
Comment (dendoesit) [reply src/client/utils/mediaQueries.ts:10]: Zak specifically asked for this one : 'Breakpoint from Desktop header to Mobile header should happen sooner (at ~<840px, currently at <800px)' but yeah we can challenge it I guess.

Comment (marian-buc) [reply src/client/utils/mediaQueries.ts:10]: it does prevent the arrow overlap, I could add it in this file only I guess

File: src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx [hunk 4]
```diff
@@ -44,9 +47,11 @@ const Wrapper = styled.div`
 const InnerWrapper = styled.div`
   margin: 0 auto;
   position: relative;
+  background: ${colorsV3.gray200};
 
-  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+  ${MEDIUM_LARGE_SCREEN_MEDIA_QUERY} {
     padding: 6.5rem 2rem;
+    background: none !important;
```
Comment (guilhermespopolin) [inline src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx:54]: Do we really need to use `!important` here?

Comment (marian-buc) [reply src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx]: nope, removed

**Review by robinandeer — APPROVED** [review]
Looks good I think :)
Haven't tested locally.

**Comment by dendoesit** [comment]
The grey200 background should only happen in Checkout Details page according to design. 
Also padding left and right issue still seems to be there. 😢 
image

**Comment by marian-buc** [comment]
added a media query @500px in ContactInformation.tsx to prevent this overflow as well:
image

**Pull request merged**